evm/deploy-amplifier-gateway.js

```diff
@@ -167,11 +168,22 @@ async function deploy(config, chain, options) {
         return;
     }
 
+    const {
+        axelar: { contracts, network, axelarId },
```

Comment:
```suggestion
        axelar: { contracts, chainId, axelarId },
```
to be consistent with other evm chain configs. Also need to add this field to all configs

Reply:
assuming `chainId` on the config, but deconstructing it still to `network` due to `chainId` being already in use within the function